deployment/get_bins.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#===============================================================================
+#          FILE: get_bins.sh
+#         USAGE: ./upgradelive.sh
+#   DESCRIPTION:
+#       OPTIONS: ---
+#  REQUIREMENTS: ---
+#          BUGS: ---
+#         NOTES: ---
+#        AUTHOR:  (), Lilach Maliniak
+#  ORGANIZATION: Kaltura, inc.
+#       CREATED: June 26, 2017
+#      REVISION:  ---
+#===============================================================================
+
+if [ "$#" -lt 1 ]; then
+	echo "usage get_bins <version> [live root path]"
+	exit 1
+fi
+
```

Comment:
i think this could be written better with for loop and a list of source and targets...